.gitignore

```diff
@@ -1 +1,2 @@
+echo coverage >> .gitignore
```

Comment:
Oh, this is from an older version of my original setup. keeping this line doesn't do anything because I didn't realize that it was actually a console command.